src/Perpetuum/Accounting/AccountManager.cs

```diff
@@ -104,6 +104,13 @@ public int GetLockedEpByAccount(Account account)
         private const double BOOSTEXPONENT = 1.5;
         private const double SERVER_DESIRED_EP_LEVEL = 500000;
 
+        private static int EPBonusBoost;
```

Comment:
In an ideal world the boost factor should be retrieved from the service providing the boost event.